.travis.yml

```
@@ -5,23 +5,24 @@ before_install:
 - docker build --target test --tag my-test-image .
 script:
 - docker run --env-file .env.test my-test-image tests/int_unit
-- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e FLASK_SKIP_DOTENV=True -e token=$token -e api_key=$api_key -e done_status=$done_status -e doing_status=$doing_status -e toDo_status=$toDo_status -e TRELLO_BOARD_ID=$TRELLO_BOARD_ID my-test-image tests/e2e
+- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e FLASK_SKIP_DOTENV=True -e MONGO_URL=$MONGO_URL -e MONGO_USER=$MONGO_USER -e MONGO_PASSWORD=$MONGO_PASSWORD -e MONGO_DB=$MONGO_DB -e MONGO_PROTOCOL=$MONGO_PROTOCOL -e MONGO_COLLECTION=$MONGO_COLLECTION my-test-image tests/e2e
```

Minor, but you could cut this statement down a little by using the shorthand that defaults to using the same variable name inside & outside the container;
```suggestion
- docker run -e FLASK_APP=todo_app/app -e FLASK_ENV=development -e FLASK_SKIP_DOTENV=True -e MONGO_URL -e MONGO_USER -e MONGO_PASSWORD-e MONGO_DB -e MONGO_PROTOCOL -e MONGO_COLLECTION my-test-image tests/e2e
```